src/chrome/content/rules/Lionbrand.com.xml

```diff
@@ -0,0 +1,11 @@
+<!--
+Refused:
```

Comment:
Maybe this line should also have some indentation?

Reply:
Sure